### PR TITLE
docs: Add v2018.2.2 release notes

### PR DESCRIPTION
**Pull request by mweinelt**
Let's get this release out asap!

@NeoRaider @rotanid @blocktrron @ecsv @T-X @bobcanthelpyou @2tata

### CODE REVIEW

File: docs/releases/v2018.2.2.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  (`#1659 <https://github.com/freifunk-gluon/gluon/issues/1659>`_)
+
+* Fixes traffic shaping not working correctly when using tunneldigger, as well as the migration between fastd 
+  and tunneldigger (`#1736 <https://github.com/freifunk-gluon/gluon/issues/1736>`_)
+
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to 4.9.18 or 4.14.128, depending on the target
```
Comment (rotanid) [inline docs/releases/v2018.2.2.rst]: 4.9.18 ? no.
4.4.128 ? no.

**Comment by rotanid** [comment]
apart from my comment: ACK

File: docs/releases/v2018.2.2.rst (hunk 1 above)
Comment (mweinelt) [reply docs/releases/v2018.2.2.rst]: 4.9.182 and 4.14.128

bd0c3988e742 kernel: bump 4.14 to 4.14.128
9de2f4d4ce kernel: bump 4.9 to 4.9.182

**Comment by rubo77** [comment]
add iinfo about the basec_races being removed in site.conf and on per default.

**Comment by bobcanthelpyou** [comment]
@rubo77 isn't that change in master, only? But not in https://github.com/freifunk-gluon/gluon/commits/v2018.2.x

File: docs/releases/v2018.2.2.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Support for the Onion Omega has been removed since the device does not
+have an ethernet port, and even with the ethernet shield connected the
+interface would not have been configured.
+
+Bugfixes
+~~~~~~~~
+
+* Fixes vulnerabilites that allowed for remote crashes and denial of service attacks through the Linux kernels
```
[the remark this answers is not in the document]
Comment (mweinelt) [reply docs/releases/v2018.2.2.rst]: Good catch!

**Pull request merged**